var/spack/repos/builtin/packages/mapl/package.py

```diff
@@ -256,9 +255,7 @@ class Mapl(CMakePackage):
     # MAPL can use ifx only from MAPL 2.51 onwards and only supports
     # ifx 2025.0 and newer due to bugs in ifx.
     conflicts("%oneapi@2025:", when="@:2.50")
-    # NOTE there is a further check on oneapi in the cmake_args below
-    # that is hard to conflict since we don't know the fortran compiler
-    # at this point
+    conflicts("^[virtuals=fortran] intel-oneapi-compilers")
```

Comment:
another place where we need to conflict with intel-parallel-studio if it's the provider as well.  Need a better way to unify that.

Reply:
:thinking: